monkestation/code/modules/slimecore/mobs/ai_controller/behaviours/clean_target.dm

```diff
@@ -1,3 +1,11 @@
+// It may be possible to merge this into `/datum/ai_behavior/execute_clean` as a subtype `/slime`.
+// However, it would require defining the cleaning behavior in
+// `/mob/living/basic/slime/resolve_unarmed_attack`, and in turn that would require finding a way to
+// reach the cleanable things from the slime mob itself.
+//
+// However, once that's done, the `/setup` and `/perform` procs can be removed, with the
+// `/finish_action` proc left intact (so as to prevent the slime from trying to use emagged cleanbot
+// phrases).
 /datum/ai_behavior/execute_clean_slime
```

Comment:
Upon further inspection, I found an issue with simply refactoring this into `execute_clean/slime`.

Specifically, the `execute_clean` routine expects a list of emagged phrases - and if none are provided, it never properly clears the target key.

I've upstreamed a fix for this at <https://github.com/tgstation/tgstation/pull/89692>, but for the moment this code will remain. When that fix is merged, I will cherry-pick it into this PR, and refactor this into `execute_clean/slime`.